versionist.conf.js

```diff
@@ -0,0 +1 @@
+module.exports = require('versionist/versionist.conf');
```

Comment:
Would https://github.com/resin-io/versionist/issues/58 make this file unnecessary?